Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/DefaultModelInfoProvider.java

```diff
@@ -53,7 +53,10 @@ public boolean accept(File path, String name) {
                     if (fileNameComponents.length == 3) {
                         Version version = new Version(fileNameComponents[2]);
                         if (requestedVersion == null || version.compatibleWith(requestedVersion)) {
-                            if (mostRecent == null || version.compareTo(mostRecent) > 0) {
+                            if (mostRecent == null ||
+                                    ((version != null && version.isComparable()) &&
+                                            (mostRecent != null && mostRecent.isComparable()) &&
+                                            version.compareTo(mostRecent) > 0)) {
                                 mostRecent = version;
```

Comment:
Same here, needs to fall back to strict equality matching if either or both are not comparable.

Reply:
Updated this. Thanks!